### PR TITLE
Add Invert Background Color buttons

### PR DESCRIPTION
**Pull request by GulsahKose**
We have that button for Writer, Calc, Draw, Impress menubar and notebookbar

Change-Id: I745c010749b8c5b88c0d181418f2a47921a94620

* Resolves: #9061
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Review by pedropintosilva — CHANGES_REQUESTED** [review]
Thanks! Please use a short label , drop `Document` so, `Invert Background`

File: browser/src/control/Control.Menubar.js [hunk 1]
```diff
@@ -440,6 +441,7 @@ L.Control.Menubar = L.Control.extend({
 				   {name: _('Show Status Bar'), id: 'showstatusbar', type: 'action'},
 				   {name: _('Hide Menu Bar'), id: 'togglemenubar', type: 'action'},
 				   {name: _('Dark Mode'), id: 'toggledarktheme', type: 'action'},
+                   {name: _('Invert Background'), id: 'invertbackground', type: 'action'},
```
Comment (timar) [inline browser/src/control/Control.Menubar.js:444]: please use tabs not spaces to indent

File: browser/src/control/Control.Menubar.js [hunk 2]
```diff
@@ -726,6 +729,7 @@ L.Control.Menubar = L.Control.extend({
...
```
Comment (timar) [inline browser/src/control/Control.Menubar.js:732]: please use tabs not spaces to indent

File: browser/src/control/Control.NotebookbarCalc.js [hunk 3]
```diff
@@ -1246,6 +1246,13 @@ L.Control.NotebookbarCalc = L.Control.NotebookbarWriter.extend({
 				'text': _('Dark Mode'),
 				'accessibility': { focusBack: true,	combination: 'DT', de: null }
 			},
+            {
```
Comment (timar) [inline browser/src/control/Control.NotebookbarCalc.js:1249]: please use tabs not spaces to indent

File: browser/src/control/Control.NotebookbarImpress.js [hunk 4]
```diff
@@ -452,6 +452,13 @@ L.Control.NotebookbarImpress = L.Control.NotebookbarWriter.extend({
 				'text': _('Dark Mode'),
 				'accessibility': { focusBack: true, combination: 'TT', de: null }
 			},
+            {
```
Comment (timar) [inline browser/src/control/Control.NotebookbarImpress.js:455]: please use tabs not spaces to indent

File: browser/src/control/Control.UIManager.js [hunk 5]
```diff
@@ -108,6 +108,10 @@ L.Control.UIManager = L.Control.extend({
 		}
 	},
 
+	invertBackground: function() {
+		// Implement this
+    },
```
Comment (timar) [inline browser/src/control/Control.UIManager.js:113]: please use tabs not spaces to indent

**Review by pedropintosilva — APPROVED** [review]

**Pull request merged**